velox/functions/prestosql/benchmarks/JsonExprBenchmark.cpp

```diff
@@ -103,28 +128,156 @@ void SIMDIsJsonScalar(int iter, int vectorSize, int jsonSize) {
   benchmark.runWithJson(iter, vectorSize, "simd_is_json_scalar", json);
 }
 
+void FollyJsonArrayContains(int iter, int vectorSize, int jsonSize) {
+  folly::BenchmarkSuspender suspender;
+  JsonBenchmark benchmark;
+  auto json = benchmark.prepareData(jsonSize);
+  suspender.dismiss();
+  benchmark.runWithJsonContains(
+      iter, vectorSize, "folly_json_array_contains", json);
+}
+
+void SIMDJsonArrayContains(int iter, int vectorSize, int jsonSize) {
+  folly::BenchmarkSuspender suspender;
+  JsonBenchmark benchmark;
+  auto json = benchmark.prepareData(jsonSize);
+  suspender.dismiss();
+  benchmark.runWithJsonContains(
+      iter, vectorSize, "simd_json_array_contains", json);
+}
+
+void FollyJsonArrayLength(int iter, int vectorSize, int jsonSize) {
+  folly::BenchmarkSuspender suspender;
+  JsonBenchmark benchmark;
+  auto json = benchmark.prepareData(jsonSize);
+  suspender.dismiss();
+  benchmark.runWithJson(iter, vectorSize, "folly_json_array_length", json);
+}
+
+void SIMDJsonArrayLength(int iter, int vectorSize, int jsonSize) {
```

Comment:
Cant we write all these 4 functions as one function that also takes the function to run (simd_json_array_length, folly_json_array_length, etc..)  something like : 
```  
 void jsonFunction(int iter, int vectorSize, int jsonSize, const char * functionName) { .. } 

 BENCHMARK_NAMED_PARAM_MULTI(jsonFunction, ...., "simd_json_array_length") ; 
 ```